### DISCUSSION TITLE
ICC profile and image size

### DISCUSSION THREAD

**Discussion by krushnavadan** [SixLabors/ImageSharp#2475] [Q&A]
Hi,
I have an issue related to the ICC profile application to a jpeg image, is it possible to create a lightweight image after the application of an ICC profile, right now saving the image with the ICC profile creates a huge image.

My original image is in KB but the ICC profile is around 12 MB, so the result is 12. Something size image, can I reduce back to kb

Thansk
KV

**Comment by JimBobSquarePants** [comment]
You could delete the profile from the metadata if it’s an sRGB one.

**Reply by JimBobSquarePants** [reply]
Otherwise we need someone to help fix #1567
